stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -851,6 +888,115 @@ internal class ChatDomainImpl internal constructor(
         return repos.selectChannelConfig(channelType)?.config ?: defaultConfig
     }
 
+    // region use-case functions
+    override fun replayEventsForActiveChannelsCall(cid: String): Call<List<ChatEvent>> = ReplayEventsForActiveChannelsImpl(this).invoke(cid)
```

Comment:
We could still use the `UseCaseHelper` internally so that we have just one instance of each use case, code might be simpler as well

Reply:
Sounds good, will do that.